docs/style_guide.adoc

```diff
@@ -96,79 +94,24 @@ g_environment: someval
 ----
 
 === Ansible Role Variables
-Ansible role variables are defined as variables contained in (or passed into) a role.
-
 '''
-[[Role-variables-MUST-have-a-prefix-of-atleast-3-characters-See.below.for.specific.naming.rules]]
+[[Ansible-role-variable-prefix]]
 [cols="2v,v"]
 |===
-| <<Role-variables-MUST-have-a-prefix-of-atleast-3-characters-See.below.for.specific.naming.rules, Rule>>
-| Role variables MUST have a prefix of at least 3 characters. See below for specific naming rules.
+| <<Ansible-role-variable-prefix, Rule>>
+| Role variables MUST have a prefix of `r_<role_name>_`
 |===
 
-==== Role with 3 (or more) words in the name
+.Definition
+Ansible variables contained in or passed into a role.
 
-Take the first letter of each of the words.
-
-.3 word example:
-* Role name: made_up_role
-* Prefix: mur
-[source]
-----
-mur_var1: value_one
-----
+.Purpose
+Name variables to avoid name collision between roles or playbooks.
```

Comment:
Do we need to have a `r_roleB_runtime` in this case or could we just update how `r_roleB_service_file` is set?

To me, if we are needing to set `r_roleB_service_file` and it is dependent on `r_roleA_runtime` then when we are calling `roleB` we would set the var like 
```yaml
r_roleB_service_file: "/etc/{{ r_roleA_runtime }}/service.file"
``` 
if we are calling `roleB` from within `roleA` otherwise we would use the variable available to us that `r_roleA_runtime` was set using...